src/hotspot/share/compiler/compileBroker.cpp

```diff
@@ -2319,9 +2319,6 @@ void CompileBroker::invoke_compiler_on_method(CompileTask* task) {
       post_compilation_event(event, task);
     }
   }
-  // Remove the JNI handle block after the ciEnv destructor has run in
-  // the previous block.
-  pop_jni_handle_block();
```

Comment:
I don't think so because most of the rest of the function is logging and it doesn't seem to do anything with JNIHandles afterwards, so there are no handles created that shouldn't be removed in that code range.